include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -600,6 +600,287 @@ struct SPhysicalDeviceLimits
+        if (minTexelOffset > _rhs.minTexelOffset) return false;
+        if (maxTexelOffset > _rhs.maxTexelOffset) return false;
+        if (minTexelGatherOffset > _rhs.minTexelGatherOffset) return false;
+        if (maxTexelGatherOffset > _rhs.maxTexelGatherOffset) return false;
+        if (minInterpolationOffset > _rhs.minInterpolationOffset) return false;
+        if (maxInterpolationOffset > _rhs.maxInterpolationOffset) return false;
+        if (maxFramebufferWidth > _rhs.maxFramebufferWidth) return false;
+        if (maxFramebufferHeight > _rhs.maxFramebufferHeight) return false;
+        if (maxFramebufferLayers > _rhs.maxFramebufferLayers) return false;
+        if (maxColorAttachments > _rhs.maxColorAttachments) return false;
+        if ((framebufferColorSampleCounts.value & _rhs.framebufferColorSampleCounts.value) != framebufferColorSampleCounts.value) return false;
+        if ((framebufferDepthSampleCounts.value & _rhs.framebufferDepthSampleCounts.value) != framebufferDepthSampleCounts.value) return false;
+        if ((framebufferStencilSampleCounts.value & _rhs.framebufferStencilSampleCounts.value) != framebufferStencilSampleCounts.value) return false;
+        if ((framebufferNoAttachmentsSampleCounts.value & _rhs.framebufferNoAttachmentsSampleCounts.value) != framebufferNoAttachmentsSampleCounts.value) return false;
+        if ((sampledImageColorSampleCounts.value & _rhs.sampledImageColorSampleCounts.value) != sampledImageColorSampleCounts.value) return false;
+        if ((sampledImageIntegerSampleCounts.value & _rhs.sampledImageIntegerSampleCounts.value) != sampledImageIntegerSampleCounts.value) return false;
+        if ((sampledImageDepthSampleCounts.value & _rhs.sampledImageDepthSampleCounts.value) != sampledImageDepthSampleCounts.value) return false;
+        if ((sampledImageStencilSampleCounts.value & _rhs.sampledImageStencilSampleCounts.value) != sampledImageStencilSampleCounts.value) return false;
+        if ((storageImageSampleCounts.value & _rhs.storageImageSampleCounts.value) != storageImageSampleCounts.value) return false;
+        if (maxSampleMaskWords > _rhs.maxSampleMaskWords) return false;
+        if (timestampComputeAndGraphics > _rhs.timestampComputeAndGraphics) return false;
+        if (timestampPeriodInNanoSeconds > _rhs.timestampPeriodInNanoSeconds) return false;
+        if (maxClipDistances > _rhs.maxClipDistances) return false;
+        if (maxCullDistances > _rhs.maxCullDistances) return false;
+        if (maxCombinedClipAndCullDistances > _rhs.maxCombinedClipAndCullDistances) return false;
+        if (discreteQueuePriorities > _rhs.discreteQueuePriorities) return false;
+        if (pointSizeRange[0] > _rhs.pointSizeRange[0]) return false;
+        if (pointSizeRange[1] > _rhs.pointSizeRange[1]) return false;
+        if (lineWidthRange[0] > _rhs.lineWidthRange[0]) return false;
+        if (lineWidthRange[1] > _rhs.lineWidthRange[1]) return false;
```

Comment:
You don't compare ranges like that, it's [minimum,maximum] of supported sizes for points/lines
We can compare them in 2 ways:
1. by length (max-min)
2. by subset (pointSizeRange should be a subset of  _rhs.pointSizeRange)
Let's go with 2

Reply:
yep lets go with 2